src/components/card/index.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+
+import styles from "./card.module.css";
+
+interface Props {
+  title: string;
+  labelText: string;
+  iconPath: string;
+  onClick: () => void;
+}
+
+export default function Card(props: Props) {
```

Comment:
```suggestion
export default function Card({title, description, iconPath, onClick}: Props) {
```

Errata: Como eu estava desconsiderando o título, eu chamei o texto de labelText. Vamos mudar e ser apenas `description`